internal/provisioner/prometheus.go

```diff
@@ -3,7 +3,7 @@ package provisioner
 import (
 	"context"
 	"fmt"
-
```

Comment:
nit: extra newline

Reply:
Wait, I'm removing a newline here @gabrieljackson  .. do you mean `strings` should be separated and line 7 should be blank?

Reply:
Oof, my bad. I hadn't had enough coffee yet. This is correct. Feel free to ignore this comment. :)